specifyweb/specify/views.py

```diff
@@ -399,86 +400,156 @@ class ReplaceRecordPT(PermissionTarget):
     update = PermissionTargetAction()
     delete = PermissionTargetAction()
 
+@transaction.atomic
+def record_merge_fx(model_name: str, old_model_id: int, new_model_id: int) -> http.HttpResponse:
+    """Replaces all the foreign keys referencing the old record ID
+    with the new record ID, and deletes the old record record.
+    """
+    # Confirm the target model table exists
+    target_model = apps.get_model('specify', model_name.lower())
+    if target_model is None:
+        return http.HttpResponseNotFound("model_name: " + model_name + "does not exist.")
+
+    # Check to make sure both the old and new agent IDs exist in the table
```

Comment:
This comment should probably be something like
`# Check to make sure both the old and new model IDs exist in the table`

Reply:
There are other cases in the comments/documentation that still reference Agent and should be changed. These occur on lines:
[425](https://github.com/specify/specify7/pull/3220/files#diff-13cfe16950398450bc9f1bf00605843b3ebbb22efca216859825059571711e1bR425)
[432](https://github.com/specify/specify7/pull/3220/files#diff-13cfe16950398450bc9f1bf00605843b3ebbb22efca216859825059571711e1bR432)
[500](https://github.com/specify/specify7/pull/3220/files#diff-13cfe16950398450bc9f1bf00605843b3ebbb22efca216859825059571711e1bR500)